packages/react-native-nitro-modules/src/Constants.ts

```diff
@@ -0,0 +1,49 @@
+import type { HybridObject } from './HybridObject'
+
+/**
+ * Converts {@linkcode property} (accessed on {@linkcode hybridObject}) to
+ * a constant on the HybridObject {@linkcode T}'s prototype.
+ *
+ * All future accesses to {@linkcode T}.{@linkcode property} will be cached on the JS side.
+ *
+ * @param hybridObject The HybridObject {@linkcode T} instance to access this property once.
+ * @param property The name of the property to convert to a constant.
+ * @example
+ * ```ts
+ * const math1 = NitroModules.createHybridObject<Math>('Math')
+ * const math2 = NitroModules.createHybridObject<Math>('Math')
+ * math1.pi // <-- accesses native code
+ * math2.pi // <-- accesses native code
+ * createConstant(math1, 'pi')
+ * math1.pi // <-- is cached on JS level
+ * math2.pi // <-- is cached on JS level
+ * ```
+ */
+export function createConstant<T extends HybridObject>(
```

Comment:
would it be helpful to add control over this JS vs native caching level?

Reply:
Well it doesn't make sense to add native caching. We want to avoid a JS -> native call, that's why we cache it in JS. That's what this function does